imagemounter/disk.py

```diff
@@ -471,6 +499,11 @@ def _mount_mmls_volumes(self):
                 volume.flag = 'unalloc'
             else:
                 volume.flag = 'alloc'
+                if ":" in slot:
+                    table, tslot = slot.split(':')
+                    volume.slot = int(table) * 4 + int(tslot)
```

Comment:
@martinvw Do you have documentation or other insights for the magic you are performing here?